### PR TITLE
docker: Use debian:bookworm-slim image, with date suffix instead of debian:stable

### PR DESCRIPTION
**Pull request by echoix**
Renovate has a special [handling of Debian versioning ](https://docs.renovatebot.com/modules/versioning/debian/) and should keep them updated with the digest pinned.

### CODE REVIEW

**Comment by echoix** [comment]
Can be merged once the checks running linters are done. The other builds don't have any impact on the outcome of docker builds that currently don't run on PRs, only once merged.

**Review by neteler — APPROVED** [review]

**Pull request merged**
